lib/internal/modules/esm/initialize_import_meta.js

```diff
@@ -47,6 +53,36 @@ function initializeImportMeta(meta, context, loader) {
   return meta;
 }
 
+/**
+ * Inspect the module URL and convert it to `__dirname` and `__filename`
+ * references if it does not point to a network module.
+ * @param {string} url
+ * @returns {{__dirname?: string, __filename?: string}}
+ */
+function resolveModuleMeta(url) {
+  if (url.slice(0, 7) !== 'file://') {
```

Comment:
```suggestion
  if (StringPrototypeSlice(url, 0, 7) !== 'file://') {
```

Reply:
Where does this `StringPrototypeSlice` come from? Is it a global that I am unaware of?

Reply:
It comes from the `primordials` object, you should add
````js
const {
  StringPrototypeSlice,
} = primordials;
````
at the top of the file, just below the `'use strict'` directive.